packages/mobile/src/screens/collection-screen/CollectionScreenDetailsTile.tsx

```diff
@@ -454,10 +453,10 @@ export const CollectionScreenDetailsTile = ({
           ddexApp={ddexApp}
           hasReposted={!!hasReposted}
           hasSaved={!!hasSaved}
-          hideFavorite={hideFavorite}
+          hideFavorite={shouldHideActions}
           hideOverflow={shouldHideOverflow}
-          hideRepost={hideRepost}
-          hideShare={hideShare}
+          hideRepost={shouldHideActions}
+          hideShare={shouldHideActions}
```

Comment:
nice! Do we still use the individual props elsewhere, or can we collapse the prop too?

Reply:
`DetailsTileActionButtons` is used in `TrackScreenDetailsTile` as well. the logic is slightly different there. honestly could be refactored tho.. but maybe not in this PR.